unitylibs/core/styles/splash-screen.css

```diff
@@ -0,0 +1,146 @@
+body > .splash-loader {
+  z-index: 11;
+}
+
+.splash-loader {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  max-height: 100%;
+  max-width: 100%;
+  color: #000;
+  z-index: 3;
+}
+
+.splash-loader .section {
+  height: 100%;
+  width: 100%;
+  max-height: 100%;
+  max-width: 100%;
+}
+
+.splash-loader.show {
+  display: flex !important;
```

Comment:
do we really need to set `!important` here? `.splash-loader.show` has higher specificity than `.splash-loader`